docker/app/Dockerfile

```diff
@@ -12,6 +12,15 @@ RUN wget -q -O /tmp/shellcheck.tar.xz "https://storage.googleapis.com/shellcheck
   && rm /tmp/shellcheck.tar.xz \
   && shellcheck --version
 
+RUN wget -q -O get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
+  && chmod 700 get_helm.sh \
+  && ./get_helm.sh \  
+  && helm version
+
+RUN wget -q https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz \
```

Comment:
Please parameterize the kubeval version (e.g. [like we're doing for shellcheck](https://github.com/ascoderu/opwen-cloudserver/blob/365b91974534f054c2503ca3a5014ae7e521eb4b/docker/app/Dockerfile#L9-L13)) and pin it to a specific version so that our build is guaranteed to be reproducible. By using latest, the build may break between two runs if a new version of kubeval gets published that has somewhat different behavior. This leads to hard to debug situations.